plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -291,34 +291,24 @@ object ScalafmtPlugin extends AutoPlugin {
 
   private lazy val sbtSources = Def.task {
     val rootBase = (LocalRootProject / baseDirectory).value
-    val thisBase = (thisProject.value).base
```

Comment:
this was already used before https://github.com/scalameta/sbt-scalafmt/commit/655b5634532ad87aa60c13b1569bb4769e051228#diff-0a93a8dada6761d84102326db6ef7b1fR260, but that commit changed a bit the implementation. @eed3si9n do you remember what was the intent and on which cases we can't rely on `rootBase` only?

Reply:
I guess it depends on where `scalafmtSbt` is scoped. If it's scoped to subprojects, then it would make sense to try to return `Nil` for all but one. If it's called once globally, then using `rootBase` only should be ok.

Reply:
>  If it's called once globally, then using rootBase only should be ok.

That's precisely what I am trying to do in this PR - scoping it only to `ThisBuild` since it does not need to aggregate across projects